pywren/wrenhandler.py

```diff
@@ -334,13 +334,18 @@ def consume_stdout(stdout, queue):
         t.start()
 
         stdout = b""
-        while t.isAlive():
+        while t.isAlive() or process.returncode is None:
```

Comment:
Are there cases `t.isAlive()` is true, and `process.returncode is None` is false?